src/lti/LTI_Constants.php

```diff
@@ -74,4 +74,18 @@ class LTI_Constants
     public const COURSE_OFFERING = 'http://purl.imsglobal.org/vocab/lis/v2/course#CourseOffering';
     public const COURSE_SECTION = 'http://purl.imsglobal.org/vocab/lis/v2/course#CourseSection';
     public const COURSE_GROUP = 'http://purl.imsglobal.org/vocab/lis/v2/course#Group';
+
+    // Context sub-roles
+    public const SUB_EXTERNALINSTRUCTOR = 'http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#ExternalInstructor';
```

Comment:
To illustrate that these are sub-roles, I added the prefix `SUB_` to each of these constants.